doc/src/base/strings.md

```diff
@@ -21,6 +21,8 @@ Base.codeunit
 Base.codeunits
 Base.ascii
 Base.@r_str
+Base.SubstitutionString
```

Comment:
AFAIK we only list exported objects here, yet `SubstitutionString` isn't exported. I guess it should be?

Reply:
`Regex` is exported even though the same objections apply, so I don't think that's a problem.